article/app/model/dotcomponents/DotcomponentsDataModel.scala

```diff
@@ -8,13 +8,14 @@ import model.dotcomrendering.pageElements.PageElement
 import navigation.NavMenu
 import play.api.libs.json._
 import play.api.mvc.RequestHeader
-import views.support.{CamelCase, GUDateTimeFormat}
+import views.support.{CamelCase, GUDateTimeFormat, ImgSrc, Item1200}
 import ai.x.play.json.Jsonx
 import common.Maps.RichMap
 import navigation.UrlHelpers.{Footer, Header, SideMenu, getReaderRevenueUrl}
 import navigation.ReaderRevenueSite.{Support, SupportContribute, SupportSubscribe}
 import model.meta.{Guardian, LinkedData, PotentialAction}
-import ai.x.play.json.implicits.optionWithNull // Note, required despite Intellij saying otherwise
+import ai.x.play.json.implicits.optionWithNull
+import common.commercial.hosted.ContentUtils // Note, required despite Intellij saying otherwise
```

Comment:
Is this import needed, + did you intend to move the comment?

Reply:
Hey I did not intend to move the comment - seems that Intellij did this 🤦‍♂️ . Thanks - will return to it's rightful place.

Reply:
I've had this happen to me too! Cool, LGTM otherwise  👍